src/components/Header.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Link } from 'gatsby';
 import AccountDropdown from './AccountDropdown';
+import logo from '../assets/banner.png';
```

Comment:
Might introduce some merge conflicts with currently in-development features, but it's the correct way of handling it